Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -3835,15 +3811,9 @@ DruidTestRTLCompiler >> gen_pushLiteralConstantBytecode [
 	"AutoGenerated by Druid"
 
 	| currentBlock |
-	self MoveR: FPReg R: TempReg.
-	self MoveM64: FoxMethod r: TempReg R: TempReg.
-	self MoveCq: LiteralStart R: ClassReg.
-	self AddCq: 1 R: ClassReg.
-	self AddCq: 8 R: TempReg.
-	self LogicalShiftLeftCq: 3 R: ClassReg.
-	self AddR: ClassReg R: TempReg.
-	self MoveM64: 0 r: TempReg R: ClassReg.
-	self ssPushRegister: ClassReg.
+	self MoveCq: ((methodObj + 8) + ((1 + LiteralStart) << 3)) R: TempReg.
+	self MoveM64: 0 r: TempReg R: TempReg.
```

Comment:
Nice! Here we should be careful because methodObj may move.

Now, an extra thing. Now these two instructions are dependent on constants.
So the Load result is also a JIT compilation constant!
This means we can fold everything into a 

`self long64At: ((methodObj + 8) + ((1 + LiteralStart) << 3))`

And knowing that that is an object (because it is a literal) we should push it AND annotate it.